pkgs/development/python-modules/shiboken6/default.nix

```diff
@@ -13,16 +13,16 @@ let
 in
 stdenv'.mkDerivation (finalAttrs: {
   pname = "shiboken6";
-  version = "6.8";
+  version = "6.8.0.2";
 
   src = fetchurl {
     # https://download.qt.io/official_releases/QtForPython/shiboken6/
     # FIXME: inconsistent version numbers in directory name and tarball?
-    url = "mirror://qt/official_releases/QtForPython/shiboken6/PySide6-${finalAttrs.version}.0-src/pyside-setup-everywhere-src-${finalAttrs.version}.tar.xz";
-    hash = "sha256-XAWtOufnJ51tudyUpG6woF/Qk1NzCfdDnDhnG9clUZA=";
+    url = "mirror://qt/official_releases/QtForPython/shiboken6/PySide6-6.8.0.2-src/pyside-setup-everywhere-src-6.8.0.tar.xz";
```

Comment:
Shouldn't we just take the first 3 digits of the version instead?

Reply:
They are exceptionally inconsistent with naming things. At this point I've just given up and hardcoded the version instead of trying to figure out the exact algorithm, since I'm pretty sure they actually just do it manually.